README.md

```diff
@@ -1,3 +1,112 @@
 # adage-server
 
 [![Code Climate](https://codeclimate.com/github/greenelab/adage-server/badges/gpa.svg)](https://codeclimate.com/github/greenelab/adage-server)
+
+## Get a working instance of the adage-server running:
+
+### Clone the adage-server repository
+```shell
+git clone git@github.com:greenelab/adage-server.git
+```
+
+### Edit settings in config.py file
```

Comment:
The `config.py` file is an extension to `settings.py` that pulls out any secrets and deployment-specific parameters, so it goes alongside `settings.py`. The idea is that you should edit `adage-server/adage/adage/config.py.template` from the repository with your deployment's information and save it as `config.py`. This puts all of the secrets and deployment-specific information into a single, easily-controlled file.

It is currently a bit noisy in `config.py.template`, but most of those parameters are used by `fab` for building a deployment. The minimal runtime sections are currently `django_key`, `databases`, and `haystack`, though as I think about it we should make `elasticsearch` connection information configurable as well. The default-supplied `haystack` works fine if `elasticsearch` is configured as specified, and (LOL) it appears you can get by with a `django_key` of "&lt;config-secret&gt;" for development purposes, so the only *essential* change is the `databases`, as @rzelayafavila has described below.